### PR TITLE
DM-39378: Rename "spuriousness" to "reliability" in the DiaSource table

### PR DESCRIPTION
**Pull request by rai-harshit**

### CODE REVIEW

File: yml/archive/baselineSchema.yaml [hunk 1]
```diff
@@ -1733,10 +1733,10 @@ tables:
       that the source is extended. extendedness = 0 implies a high degree of confidence
       that the source is point-like.
     mysql:datatype: FLOAT
-  - name: spuriousness
```
Comment (parejkoj) [inline yml/archive/baselineSchema.yaml:1736]: Please revert all changes to this file: files in `archive/` are there for historical reference and should not be modified.

**Review by parejkoj — COMMENTED** [review]
This, and all the other tickets on this branch, should be merged into DM-37196, I believe.

File: yml/archive/baselineSchema.yaml (hunk 1 above)
Comment (rai-harshit) [reply yml/archive/baselineSchema.yaml]: Reverted the changes.

**Comment by rai-harshit** [comment]
> This, and all the other tickets on this branch, should be merged into DM-37196, I believe.

Here is an update:
Successful changes- `alert_packet`, `sdm_schemas`
Merge conflicts- `ap_association` (I resolved the conflicts, but could you please verify it once?)
No corresponding 37196 ticket- `ap_verify`, `analysis_ap`

**Review by parejkoj — APPROVED** [review]

**Pull request merged**